Issue1〜3の修正

masayo-ikeda opened the pull request:
Issue1〜3の解消

masayo-ikeda committed:
Issue1

masayo-ikeda committed:
Issue1

masayo-ikeda committed:
Issue2

masayo-ikeda committed:
Revert "Issue2"

This reverts commit ce7930c929080f4483d01a781370ad1845ec07a9.

masayo-ikeda committed:
Issue2

masayo-ikeda committed:
Issue3

shuki-yamamoto commented on app/views/calendars/index.html.erb:
```diff
@@ -14,13 +14,16 @@
         <div class='date'>
           <% day[:month] %>/<% day[:date] %>
         </div>
```
index.html.erbのファイル内に不要なdivタグが残っています。
class名が付与されていない等のdivタグが無いか見直していただき修正をお願い致します。

masayo-ikeda replied:
ご指摘を頂きありがとうございます、再度修正します、宜しくお願いします

bachica replied:
まだ不要なdivタグが残っております！
クラス名のないdivタグが２ペア残っていますので探してみてください！

shuki-yamamoto reviewed (commented):
大変お待たせいたしました。
issue1~3のレビュー依頼ありがとうございます。

1点コメントいたしましたので、ご確認をお願いいたします。
また、記載頂いているコメントアウトですが、もし必要であれば、第三者でもわかるように「残す目的」「なぜ必要か」という理由などをコメントとして書いておきましょう！

以上、宜しくお願い致します。

masayo-ikeda committed:
修正箇所Issue3再

不要なdivタグを消去しました
理由はclassはdateを指しており
monthは指していませんでした
よって、必要なdateのみ表示したいので今回の実装になります

bachica commented on app/views/calendars/index.html.erb:
```diff
@@ -12,15 +12,18 @@
         </div>
-        <ul class='content'>
+        <%# <ul class='content'> %>
+        <li class='content'>
```
ul要素をli要素に変更（またはその逆）された箇所がいくつかあるようですが、
これらは変更不要のため元の記述に戻しておくようお願いいたします！

masayo-ikeda replied:
ご指摘ありがとうございます、元に戻しました

bachica commented on app/views/calendars/index.html.erb:
```diff
@@ -12,15 +12,18 @@
       <div class='item'>
         <div class='date'>
-          <% day[:month] %>/<% day[:date] %>
+          <% day[:date] %>
```
`<% day[:date] %>　　→　　<% day[:month] %>/<% day[:date] %>`

こちらの記述は変更不要のため、元の記述に戻しておきましょう

masayo-ikeda replied:
ご指摘ありがとうございました、元に戻しました

bachica reviewed (commented):
レビュー依頼ありがとうございます！
修正点確認させていただきました。
追加で３点コメントいたしましたので、ご確認お願いいたします！


不明なことがあればトレーナーにお気軽にご相談ください！

修正後、再度レビュー依頼フォームをご提出ください。
よろしくお願いいたします！

masayo-ikeda committed:
修正箇所Issue3再2

指摘があった２箇所を
元に戻しました
さらに、不要なdivタグ消去しました
理由は、無関係なdivタグの為どこにも関連しないからです

shuki-yamamoto reviewed (commented):
レビュー依頼ありがとうございます！
LGTMとさせて頂きます、issue1~3の実装お疲れ様でした！
masterブランチにfirst_trainingブランチをマージ後、次の実装に取り組みましょう！
応援しております！
